test/Sentry.Tests/HubTests.cs

```diff
@@ -1730,6 +1730,54 @@ public void CaptureFeedback_ConfigureScope_ScopeApplied(bool enabled)
         _fixture.Client.Received(enabled ? 1 : 0).CaptureFeedback(Arg.Any<SentryFeedback>(), Arg.Is<Scope>(s => s.Tags["foo"] == "bar"), Arg.Any<SentryHint>());
     }
 
+    [Theory]
+    [InlineData(true)]
+    [InlineData(false)]
+    public void CaptureAttachment_HubEnabled(bool enabled)
+    {
+        // Arrange
+        var hub = _fixture.GetSut();
+        if (!enabled)
+        {
+            hub.Dispose();
+        }
+
+        _fixture.Client.CaptureEnvelope(Arg.Any<Envelope>()).Returns(true);
```

Comment:
thought: could/should we Assert that `CaptureEnvelope` actually was called in case the Hub is enabled, and was _not_ called in the case the Hub is disabled?